docs-gb/kubernetes/hpa-rps-autoscaling.md

```diff
@@ -246,29 +276,111 @@ spec:
         averageValue: 3
 ```
 
-In the two HPA manifests above, the scaling metric is exactly the same, and uses the exact same parameters: this is to ensure that both the Models and the Servers are scaled up/down at approximately the same time. Small variations in the scale-up time are expected because each HPA samples the metrics independently, at regular intervals. If a Model gets scaled up slightly before its corresponding Server, the model is currently marked with the condition ModelReady "Status: False" with a "ScheduleFailed" message until new Server replicas become available. However, the existing replicas of that model remain available and will continue to serve inference load.
+In the two HPA manifests above, the scaling metric is exactly the same, and uses the exact same
```

Comment:
```suggestion
In the preceding HPA manifests, the scaling metric is exactly the same, and uses the exact same
```
Don't use directional language to orient the reader, such as above, below, or right-hand side. This type of language doesn't work well for accessibility or for localization reasons.

Don't use directional language to refer to a position in a document. Instead, use earlier, preceding, or following.

Recommended: In the preceding diagram, clients run jobs on multi-team or single-team clusters.

Not recommended: In the diagram above, clients run jobs on multi-team or single-team clusters.